frontend/src/routes/manage/Realm/Content/Edit/MoveButtons.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { useFragment, graphql, useMutation } from "react-relay";
+import { FiArrowDown, FiArrowUp } from "react-icons/fi";
+
+import type { MoveButtonsData$key } from "../../../../../query-types/MoveButtonsData.graphql";
+import type { MoveButtonsMutation } from "../../../../../query-types/MoveButtonsMutation.graphql";
+import { Button } from "../util";
+
+
+type Props = {
+    realm: MoveButtonsData$key;
+    index: number;
+    onCommit?: () => void;
+    onCompleted?: () => void;
+    onError?: (error: Error) => void;
+};
+
+export const MoveButtons: React.FC<Props> = ({
+    realm,
+    index,
+    onCommit,
+    onCompleted,
+    onError,
+}) => {
+    const { t } = useTranslation();
+
+
+    const { id: realmId, blocks } = useFragment(graphql`
+        fragment MoveButtonsData on Realm {
+            id
+            # We need this list only for the length,
+            # but we have to query *something* from it.
+            blocks { id }
```

Comment:
I was already thinking whether passing the values directly (instead of using fragments) would be easier in this whole "edit content blocks" thing. Dunno :shrug:

Reply:
Yes maybe, but I'm not gonna make that refactor now anymore unless you ask really really really really nicely. :stuck_out_tongue_winking_eye: 

But yes, trying to consistently use the Relay idea in these components was a major pain. I would even go so far, though, as to say that querying everything on top and just passing it through wouldn't be any better. You would still have to decide what to pass every time, and it is the fact that this decision was often wrong that was giving me a lot of trouble. In the future I think I would just pass the whole damn object around or even make a context. Yes, something something coupling, but it's not like the individual subcomponents of this page will see much reuse ... They are highly coupled anyway. :shrug:

Reply:
Yeah mh interesting. Probably a discussion for another time, but you are right: not in this PR.